src/web/src/components/Posts/Post.tsx

```diff
@@ -320,90 +321,92 @@ const PostComponent = ({ postUrl, currentPost, totalPosts }: Props) => {
   }
 
   return (
-    <Box className={classes.root}>
-      {currentPost && totalPosts && (
-        <div className={classes.postCount}>
-          <Chip
-            label={`${currentPost.toLocaleString()} of ${totalPosts.toLocaleString()}`}
-            variant="outlined"
-            className={classes.chipComponent}
-          />
-        </div>
-      )}
+    <GithubInfoProvider urls={extractedGitHubUrls}>
```

Comment:
Instead of passing the URLs into this, why don't we pass the raw string content of the post, and move the URL extraction/parsing into the hook?  It will reduce the size of this component, and isolate that logic in one spot.